helix_fhir_client_sdk/test/test_patch_mixin.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+import pytest
+from typing import Dict, Any
+
+from aioresponses import aioresponses
+
+from helix_fhir_client_sdk.fhir_client import FhirClient
+
+
+@pytest.mark.asyncio
+async def test_patch_async_success() -> None:
```

Comment:
@imranq2 just wanted to have a unit test for the PATCH op here
but this test actually failed when I ran local,
the error was
```
E           aiohttp.client_exceptions.ClientConnectionError: Connection refused: PATCH http://example.com/Observation/1
/usr/local/lib/python3.12/site-packages/aioresponses/core.py:515: ClientConnectionError
```

any help would be much appreciated. thanks